### PR TITLE
PE-1008 : refactor/handle unknown upload errors

### PR DESCRIPTION
**Pull request by jdaev**
Updates arweave-dart to handle uncaught errors in the upload chunk post request.

### CODE REVIEW

File: lib/blocs/drive_detail/drive_detail_cubit.dart [hunk 1]
```diff
@@ -62,13 +62,11 @@ class DriveDetailCubit extends Cubit<DriveDetailState> {
...
     await _folderSubscription?.cancel();
     // For attaching drives. If drive is not found, emit state to prompt drive attach
-    await _driveDao.driveById(driveId: driveId).getSingleOrNull().then((value) {
-      if (value == null) {
-        emit(DriveDetailLoadNotFound());
-        return;
-      }
-    });
-
+    final drive = await _driveDao.driveById(driveId: driveId).getSingleOrNull();
```
Comment (jdaev) [inline lib/blocs/drive_detail/drive_detail_cubit.dart:65]: This is just to stop an uncaught exception in the drive attach flow that slipped through. Unrelated but an easy win.

File: macos/Flutter/GeneratedPluginRegistrant.swift [hunk 2]
```diff
@@ -12,7 +12,7 @@ import url_launcher_macos
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
   FileSelectorPlugin.register(with: registry.registrar(forPlugin: "FileSelectorPlugin"))
-  FLTPackageInfoPlugin.register(with: registry.registrar(forPlugin: "FLTPackageInfoPlugin"))
+  FLTPackageInfoPlusPlugin.register(with: registry.registrar(forPlugin: "FLTPackageInfoPlusPlugin"))
```
Comment (jdaev) [inline macos/Flutter/GeneratedPluginRegistrant.swift:15]: macos platform file, autogenerated by flutter due to version update.

**Review by fedellen — DISMISSED** [review]

**Review by karlprieb — DISMISSED** [review]

File: pubspec.lock [hunk 3]
```diff
@@ -47,11 +47,11 @@ packages:
...
     description:
       path: "."
-      ref: "27fea131c7e5c95c7b4e0e632d9618cb6ac9d856"
-      resolved-ref: "27fea131c7e5c95c7b4e0e632d9618cb6ac9d856"
+      ref: bbe586a21608e520a68ce54ac95831aa2e1e877a
+      resolved-ref: bbe586a21608e520a68ce54ac95831aa2e1e877a
       url: "https://github.com/ardriveapp/arweave-dart"
     source: git
-    version: "0.1.0"
+    version: "2.0.0"
```
Comment (arielmelendez) [inline pubspec.lock:54]: Use the correct version once we get the arweave-dart tags sorted out (probably 2.0.1).

**Review by arielmelendez — CHANGES_REQUESTED** [review]
See tag number comment

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**